hack/lib/util.sh

```diff
@@ -446,6 +446,9 @@ kube::util::ensure_godep_version() {
 
   kube::log::status "Installing godep version ${GODEP_VERSION}"
   go install ./vendor/github.com/tools/godep/
+  GP="$(echo $GOPATH | cut -f1 -d:)"
+  hash -r # force bash to clear PATH cache
+  PATH="${GP}/bin:${PATH}"
```

Comment:
I really don't think util scrips should be changing PATH like this.  It should be the caller's responsibilityt o make sure GOPATH/bin is in PATH

Reply:
https://github.com/kubernetes/kubernetes/pull/57600/files#diff-fa58360a7125fddab9491c80bb3f1a87

and

https://github.com/kubernetes/kubernetes/pull/57600/files#diff-85254102210f2a3edc79a4ae67a3448e

Reply:
This was previously in the script, before your PR (#58975) to vendor godep. The issue is not just it being *in* the path, but also being at the *front* of the path so that we select it first. Note, we aren't exporting this, so it's not persistent past the end of the script.

Reply:
The other option I considered here was setting a temp GOBIN with a well-known location, then calling it directly. I opted for the option that was closer to the revert because this was blocking all presubmits triggered godep verification.

Reply:
> It should be the caller's responsibilityt o make sure GOPATH/bin is in PATH

+1